src/scenic/core/regions.py

```diff
@@ -1630,17 +1677,17 @@ def intersect(self, other, triedReversed=False):
         if isinstance(other, PolygonalFootprintRegion):
             # Other region is a PolygonalFootprintRegion, so we can just intersect the base polygons
             # and take the footprint of the result, if it isn't empty.
-            new_poly = PolygonalRegion(polygon=self.polygon).intersect(PolygonalRegion(polygon=other.polygon))
+            new_poly = self.polygons.intersect(other.polygons)
```

Comment:
I guess we have no tests for this case, because there is no `intersect` method on shapely polygons (presumably you meant `intersection`). Better add one.

Reply:
Ah yes good catch, those tests are on the list.